src/images/desktop/pocket-casts.svg

```diff
@@ -0,0 +1 @@
8-1.903 5.426-4.266 5.426m-.021-8.36c-.901 0-1.462.966-1.462 2.934 0 1.99.581 2.974 1.502 2.974.901 0 1.482-.984 1.482-2.953 0-1.99-.601-2.954-1.522-2.954M464.006 22.267c-2.784 0-4.427-1.95-4.427-5.345 0-3.316 1.683-5.406 4.587-5.406.922 0 1.622.18 2.303.443l-.441 2.532c-.48-.222-1.04-.403-1.622-.403-1.342 0-2.043 1.166-2.043 2.793 0 1.769.701 2.814 2.084 2.814.52 0 1.06-.161 1.641-.382l.441 2.432a6.654 6.654 0 01-2.523.522M470.474 17.123v4.984h-2.905V7.618h2.905v8.58l2.363-4.5h3.184l-2.824 4.982 2.905 5.427h-3.225l-2.403-4.984" fill="#231F20" fill-rule="nonzero"/><path d="M475.718 16.941c0-3.576 1.824-5.465 4.128-5.465 2.443 0 4.086 1.868 4.086 5.526v.944h-5.51c.102 1.427.883 2.13 2.164 2.13.762 0 1.663-.241 2.484-.622l.52 2.05a7.528 7.528 0 01-3.304.763c-2.544 0-4.568-1.346-4.568-5.326zm5.53-.743c-.06-1.567-.522-2.49-1.402-2.49-.903 0-1.363.963-1.423 2.49h2.825M485.491 18.99v-4.902h-1.12v-2.391h1.12V8.965h2.885v2.732h2.203v2.391h-2.203v4.482c0 .863.46 1.225 1.241 1.225.341 0 .642-.06.962-.14v2.37c-.441.14-1.062.222-1.782.222-2.203 0-3.306-.904-3.306-3.256M500.331 22.267c-3.645 0-5.93-2.33-5.93-7.174 0-4.661 2.345-7.214 5.95-7.214 1.202 0 2.204.241 2.985.543l-.481 2.854c-.62-.261-1.342-.503-2.203-.503-1.922 0-3.065 1.527-3.065 4.28 0 2.833 1.082 4.32 3.125 4.32.78 0 1.522-.2 2.163-.482l.5 2.773a7.978 7.978 0 01-3.044.603M506.698 22.247c-1.622 0-2.763-1.045-2.763-3.296 0-1.005.28-1.847.821-2.371.58-.603 1.442-.904 2.743-.904.562 0 1.122.08 1.603.181v-.583c0-.884-.401-1.366-1.482-1.366-.841 0-1.743.24-2.544.583l-.54-2.21c1.041-.422 2.263-.744 3.604-.744 2.624 0 3.806 1.124 3.806 3.516v5.205l.22 1.849h-2.523l-.36-.905c-.762.663-1.503 1.045-2.585 1.045m2.404-2.35v-2.473c-.28-.06-.601-.08-.941-.08-.601 0-.941.12-1.182.341-.22.242-.34.603-.34 1.126 0 1.044.46 1.466 1.26 1.466.402 0 .822-.16 1.203-.38M516.772 22.287a8.059 8.059 0 01-3.827-.965l.581-2.35c1.103.602 2.203.964 3.205.964.742 0 1.142-.321 1.142-.764 0-.441-.24-.643-1.642-1.205-1.943-.784-2.905-1.527-2.905-3.276 0-1.93 1.322-3.175 3.646-3.175 1.18 0 2.263.282 3.244.765l-.58 2.35c-.902-.462-1.823-.764-2.664-.764-.62 0-.962.262-.962.684 0 .402.22.602 1.603 1.166 2.184.863 2.944 1.667 2.944 3.315 0 2.17-1.522 3.255-3.785 3.255M522.218 18.99v-4.902h-1.122v-2.391h1.122V8.965h2.884v2.732h2.203v2.391h-2.203v4.482c0 .863.461 1.225 1.241 1.225.341 0 .641-.06.962-.14v2.37c-.44.14-1.062.222-1.783.222-2.203 0-3.304-.904-3.304-3.256" fill="#231F20" fill-rule="nonzero"/><g transform="translate(527.939 11.28)"><mask id="c" fill="#fff"><use xlink:href="#b"/></mask><path d="M4.072 11.006a8.059 8.059 0 01-3.826-.965l.58-2.35c1.103.602 2.204.965 3.205.965.741 0 1.143-.322 1.143-.764s-.241-.644-1.643-1.206C1.588 5.903.627 5.16.627 3.411.627 1.48 1.949.236 4.272.236c1.181 0 2.263.281 3.244.764l-.58 2.35c-.902-.462-1.823-.764-2.664-.764-.62 0-.961.263-.961.685 0 .401.22.602 1.602 1.165 2.183.863 2.943 1.667 2.943 3.315 0 2.17-1.52 3.255-3.784 3.255" fill="#231F20" fill-rule="nonzero" mask="url(#c)"/></g></g></svg>
```

Comment:
It is good idea to always minify SVG files before adding it to repo (final build is lighter). Not sure if those icons could be minified, but you can try.

This is good tool: https://jakearchibald.github.io/svgomg/ (default settings)